process/process.go

```diff
@@ -64,6 +64,7 @@ type Config struct {
 	Path            string
 	Args            []string
 	Env             []string
+	Stdin           io.Reader
```

Comment:
Here's the lowest layer learning to take Stdin — our `process.Config`.
The zero-value of io.Reader is nil, so when it's not passed it's the same as the old `p.command.Stdin = nil` code.